lib/instrumentation/modules/@elastic/elasticsearch.js

```diff
@@ -73,12 +74,13 @@ module.exports = function (elasticsearch, agent, { version, enabled }) {
           return
         }
 
-        setElasticsearchDbContext(span, request.params.path,
-          request.params.querystring, request.params.body)
+        setElasticsearchDbContext(span, paramPath, paramQueryString, paramBody)
 
-        const { hostname, port } = result.meta.connection.url
-        span.setDestinationContext(
-          getDBDestination(span, hostname, port))
+        if (result.meta.connection && result.meta.connection.url) {
```

Comment:
nit: here, and elsewhere, we're still assuming that `result.meta` is a set value, which means we're still open to future client change causing another crashing error. (A moot point if we get our friends to incorporate the agent in their integration tests, ergo `nit`)

Reply:
Yah, fair point. I'll give it a try. I guess I'll have to [wait for node v14 for `foo?.bar?.baz`](https://node.green/#ES2020-features-optional-chaining-operator------optional-property-access).

Reply:
Done in commit f8d3f516